src/server/routes/transaction/blockchain/getLogs.ts

```diff
@@ -0,0 +1,189 @@
+import { Static, Type } from "@sinclair/typebox";
+import { FastifyInstance } from "fastify";
+import { StatusCodes } from "http-status-codes";
+import {
+  eth_getTransactionReceipt,
+  getContract,
+  getRpcClient,
+  parseEventLogs,
+  prepareEvent,
+} from "thirdweb";
+import { resolveContractAbi } from "thirdweb/contract";
+import { getChain } from "../../../../utils/chain";
+import { thirdwebClient } from "../../../../utils/sdk";
+import { createCustomError } from "../../../middleware/error";
+import { standardResponseSchema } from "../../../schemas/sharedApiSchemas";
+import { getChainIdFromChain } from "../../../utils/chain";
+
+declare global {
+  interface BigInt {
+    toJSON: () => string;
+  }
+}
+BigInt.prototype.toJSON = function () {
+  return `${this}n`;
+};
+
+// INPUT
+const requestSchema = Type.Object({
+  txHash: Type.String({
```

Comment:
1. Spell our `transactionHash` or just `hash`
2. 👍 for the validation here. Can you move this to a shared `TransactionHashSchema`